### PR TITLE
[86by8j89h][docs] Add Flags a11y guide and example

### PR DESCRIPTION
**Pull request by sheila-semrush**
## Motivation and Context

1. Added an a11y recommendation that `aria-label` and `role` should be used when using `Flags` without the full country name.
2. Added a corresponding example right to the A11y tab — I think it's not worth complicating this guide's structure while it still doesn't have that much information.

**IMPORTANT.** This PR already assumes that `Flags` doesn't have an `aria-label`, so it should be merged only when the `aria-label` is actually removed from the component.

## How has this been tested?

Manually.

## Screenshots:

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: website/docs/components/flags/examples/aria-label.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const Demo = () => (
+  <>
+    {countries.map((country) => (
+      <Flags
+        key={country}
+        name={country as keyof typeof iso2Name}
+        role='image'
+        aria-label={iso2Name[country]}
+        mr={1}
```
Comment (j-mnizhek) [inline website/docs/components/flags/examples/aria-label.tsx:16]: I would suggest to add mb={3} here just to visually separate first row with the following column a little bit. Something like this:
image

Comment (j-mnizhek) [reply website/docs/components/flags/examples/aria-label.tsx:16]: Great example btw!

**Review by j-mnizhek — APPROVED** [review]
Added small suggestion. Everything other is great in my opinion ✨

File: website/docs/components/flags/examples/aria-label.tsx (hunk 1 above)
Comment (sheila-semrush) [reply website/docs/components/flags/examples/aria-label.tsx:16]: Added the margin, thanks!

**Review by msereniti — APPROVED** [review]

**Pull request merged**